_data/maintainers.yml

```
@@ -518,6 +518,10 @@ kdamodaran1:
   label: kdamodaran1
   url: https://github.com/kdamodaran1
   name: kdamodaran1
+kgpayne:
+  label: kgpayne
```

Since we know this guy. 😄 

```suggestion
  label: Ken Payne
```